include/crm/crm.h

```diff
@@ -187,6 +187,7 @@ extern char *crm_system_name;
 #  define RSC_NOTIFIED	CRMD_ACTION_NOTIFIED
 
 #  define RSC_STATUS	CRMD_ACTION_STATUS
+#  define RSC_PROBE	CRMD_ACTION_STATUS"(probe)"
```

Comment:
would leave a space after `CRMD_ACTION_STATUS` as tokens are pasted together automatically by the preprocessor, regardless of space